applications/content-drafting.md

```diff
@@ -0,0 +1,29 @@
+Machine translation allows to quickly draft content in natural language. Applications:
+
+# Mock-up localization
+Raw machine-translated text is a good mock-up to verify how the target language appears in localized content such as:
+* user interface,
```

Comment:
For lists without verbs, we don't use punctuation.

[Lists](https://machinetranslate.org/style#lists)

Reply:
Done